gpu4pyscf/lib/gvhf-rys/rys_roots.cu

```diff
@@ -0,0 +1,72 @@
+#include "rys_roots.cuh"
```

Comment:
Is this file the same as rys_roots_unrolled.cu?

Reply:
No. different ways to assign threads.